install.sh

```diff
@@ -132,9 +135,13 @@ if [ "$(uname)" = "Linux" ]; then
     sudo apt-get update
     sudo apt-get install -y python3.8-venv python3-distutils openssl
   elif [ "$UBUNTU_2100" = "1" ]; then
-    echo "Installing on Ubuntu 21.04 or newer."
+    echo "Installing on Ubuntu 21.04."
```

Comment:
I think this could also be 21.10? Should this be "Intalling on Ubuntu 21.x" or something along those lines?

Reply:
This continues the existing pattern.  I think we can take up addressing all of these version claims separately.